config/m4/sysdep.m4

```diff
@@ -104,15 +104,30 @@ AC_CHECK_HEADERS([sys/platform/ppc.h])
 #
 # PowerPC query for getting TB frequency
 #
-AC_CHECK_DECLS([__ppc_get_timebase_freq], [], [], [#include <sys/platform/ppc.h>])
-AC_CHECK_FUNCS([__ppc_get_timebase_freq])
-
-
-#
-# PowerPC query for getting TB
-#
-AC_CHECK_DECLS([__ppc_get_timebase], [], [], [#include <sys/platform/ppc.h>])
-AC_CHECK_FUNCS([__ppc_get_timebase])
+AC_CHECK_DECL([__ppc_get_timebase_freq],
+              [AC_CHECK_FUNCS([__ppc_get_timebase_freq])], [],
+              [#include <sys/platform/ppc.h>])
+
+
+#
+# PowerPC query for getting TB.
+# Note: AC_CHECK_FUNCS doesn't work for checking __ppc_get_timebase()
+#
+AC_COMPILE_IFELSE([AC_LANG_SOURCE([[
+                    #include <sys/platform/ppc.h>
+
+					int main(int argc, char** argv) {
+						__ppc_get_timebase();
+						return 0;
+					} ]])],
```

Comment:
why check_funcs not enough?
maybe need ac_link_ifelse?

Reply:
also indent seems wrong

Reply:
> maybe need ac_link_ifelse?

done

> also indent seems wrong

fixed